tools/generate_changelog.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+import sys
+
+from validator.checker import UpdateChecker
+from validator import get_latest_version
+
+root_dir = Path(__file__).parents[1]
+
+if __name__ == '__main__':
```

Comment:
I am getting a dejavu here. Use argparse (https://docs.python.org/3/library/argparse.html) and don't read the `sys.argv` directly

Reply:
😅😅😅😅😅☠️